test/support/admin_edition_controller_test_helpers.rb

```diff
@@ -1298,42 +1298,6 @@ def should_allow_association_with_related_mainstream_content(edition_type)
       end
     end
 
-    def should_allow_alternative_format_provider_for(edition_type)
```

Comment:
Why has this test been removed? 🤔 

Has the new Design System done away with the 'alternative format provider' option on documents?

Reply:
Yea good spot. Sorry this was a merge conflict issue!